errtracker/errtracker.go

```diff
@@ -79,6 +82,14 @@ func readMachineID() ([]byte, error) {
 	return nil, fmt.Errorf("cannot report: no suitable machine id file found")
 }
 
+func snapConfineProfileDigest(suffix string) string {
+	profileText, err := ioutil.ReadFile(snapConfineProfile + suffix)
+	if err != nil {
+		return ""
+	}
+	return fmt.Sprintf("%x", sha1.Sum(profileText))
```

Comment:
If we use md5 here we can more easily compare with the dpkg metadata. This way we can look at e.g. `dpkg -s snapd` and immediately see the md5sums of the conffiles without having to unpack the debs.